src/lib/components/modals/ProjectModal.svelte

```diff
@@ -100,7 +100,7 @@
                     <enhanced:img
                         src={currentItem.picture}
                         alt="dummy"
-                        loading="lazy"
+                        loading="eager"
```

Comment:
```suggestion
                        loading="lazy"
```



In retrospect, I think `lazy` is still the right way to go here. Viewers don't necessarily see all images, so loading them at the start isn't particularly necessary and only reduces initial page load times.